subsystem/utils_test.go

```diff
@@ -26,6 +26,19 @@ const (
 )
 
 func clearDB() {
+	if !Options.EnableKubeAPI {
+		// Delete cluster should use the REST API in order to delete any
+		// clusters' resources managed by the service
+		reply, err := userBMClient.Installer.ListClusters(context.Background(), &installer.ListClustersParams{})
+		Expect(err).ShouldNot(HaveOccurred())
+		for _, c := range reply.GetPayload() {
+			// DeregisterCluster API isn't necessarily available (e.g. cluster is being installed)
+			// so the returned value isn't verified
```

Comment:
suggestion: catch the error result and add a debug log to be printed in case there was such an error.
with that, we'll at least have some visibility in case we'll need it.